server/sonar-server/src/main/java/org/sonar/server/test/ws/ListAction.java

```diff
@@ -185,15 +194,8 @@ private static int coveredLines(List<CoveredFileDoc> coveredFiles) {
     return Maps.uniqueIndex(components, ComponentDtoFunctions.toUuid());
   }
 
-  private static class TestToFileUuidFunction implements Function<TestDoc, String> {
-    @Override
-    public String apply(@Nonnull TestDoc testDoc) {
-      return testDoc.fileUuid();
-    }
-  }
-
   private SearchResult<TestDoc> searchTests(DbSession dbSession, @Nullable String testUuid, @Nullable String testFileUuid, @Nullable String testFileKey,
```

Comment:
It would be interesting to creating a class like a TestsFields that would be created from the request, and that would return all you need : testFileUuid, testFileKey, sourceFileUuid, etc.
This class would also have the responsability to check arguments.